tests/sles4sap/redirection_tests/hana_cluster_check.pm

```diff
@@ -37,6 +37,7 @@ sub run {
             check_cluster_state();
 
             # saputils lib
+            # ( After move to 'lib/sles4sap_publiccloud.pm' this need to be changed to use 'get_hana_topology()' )
```

Comment:
Please remove this comment. 
`get_hana_topology` is a public cloud library based module and won't work with on premise type of test.

Reply:
Done.